src/gossip/data.zig

```diff
@@ -335,7 +250,7 @@ pub const CrdsValueLabel = union(enum) {
     ContactInfo: Pubkey,
 };
 
-pub const CrdsData = union(enum(u32)) {
+pub const GossipData = union(enum(u32)) {
```

Comment:
I'm fine with "GossipData", but for posterity I'd just like to publish what I said already in person.

I think the name is a little ambiguous because "data" is very generic and could refer to different types of information. Also, technically, "data" is plural, so it feels weird to use it to refer to a single data record.

The best alternative I could think of is "GossipRecord" because to me "record" implies something like a row in a database. It represents the idea of a single data entry that is stored as-is in some storage medium. So it feels like a more precise way to communicate what this type represents.

That said I think it's pretty clear that GossipData represents a single record as soon as you see the type definition.